lnbits/core/crud.py

```diff
@@ -766,9 +767,11 @@ async def get_payments_paginated(
     if complete and pending:
         pass
     elif complete:
-        clause.append("((amount > 0 AND pending = false) OR amount < 0)")
+        clause.append(
+            f"((amount > 0 AND status NOT '{PaymentState.PENDING}') OR amount < 0)"
```

Comment:
Should be a positive inclusion condition rather than an exclusion one.
What happens with the `FAILED`, should they be included here?
What if we introduce a different `PENDING`-like status (eg `IN_FLIGHT`)?

```suggestion
            f"((amount > 0 AND status IN (<all complete statues here>)) OR amount < 0)"
```